src/metrics/master.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "metrics/metrics.h"
+
+#ifdef HAVE_PROMETHEUS
+#include <prometheus/counter.h>
+#include <prometheus/registry.h>
+
+namespace metrics::master {
+
+struct Master {
+public:
+	Master() = default;
+	Master(std::shared_ptr<prometheus::Registry>& registry);
+
+	// Metric(s)
+	prometheus::Family<prometheus::Counter> *packet_client_counter{nullptr};
```

Comment:
Maybe `prometheus::Family<prometheus::Counter>` can be defined as a custom data type with a shorter name by using C++ `using ` instruction and then use this data type in all functions.

```cpp
using MetricsCounter = prometheus::Family<prometheus::Counter>;

// Metric(s)
MetricsCounter *packet_client_counter{nullptr};
MetricsCounter *byte_client_counter{nullptr};
MetricsCounter *filesystem_counter{nullptr};
MetricsCounter *chunk_counter{nullptr};
```